ci-operator/step-registry/observers/resource-watch/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- deads2k
+- dgoodwin
+- stbenjam
```

Comment:
Please feel free to add yourself here, we'll need your expertise to approve having dug into this.